pkg/kapis/devops/v1alpha3/register.go

```diff
@@ -64,6 +66,10 @@ func AddToContainer(container *restful.Container, devopsClient devopsClient.Inte
 func registerRoutes(devopsClient devopsClient.Interface, k8sClient k8s.Client, client client.Client, ws *restful.WebService) {
 	handler := newDevOpsHandler(devopsClient, k8sClient)
 	registerRoutersForCredentials(handler, ws)
+	registerRoutersForCredentials(handler, ws)
```

Comment:
Duplicate registering routers for credentials, please remove it.

Reply:
Sure.